app/views/idv/in_person/state_id.html.erb

```diff
@@ -3,7 +3,11 @@
 <% if updating_state_id %>
   <%= render PageHeadingComponent.new.with_content(t('in_person_proofing.headings.update_state_id')) %>
 <% else %>
-  <%= render PageHeadingComponent.new.with_content(t('in_person_proofing.headings.state_id')) %>
+  <% if IdentityConfig.store.in_person_capture_secondary_id_enabled %>
+    <%= render PageHeadingComponent.new.with_content(t('in_person_proofing.headings.state_id_milestone_2')) %>
+  <% else %>
+    <%= render PageHeadingComponent.new.with_content(t('in_person_proofing.headings.state_id')) %>
+  <% end %>
```

Comment:
Wish this would have come up in refinement instead of PR but I'd prefer if we just update the page heading for all users -- this change doesn't seem like it needs to be restricted to DAV only. Conditional content adds overhead to PR reviews, product acceptance, bug bashing, etc so I'd prefer to ask design if we can reduce it when possible. In this case [Kelli thinks we can just update it for all users](https://gsa-tts.slack.com/archives/C03FA4VBN76/p1678821813290069?thread_ts=1678821760.157809&cid=C03FA4VBN76)

Reply:
Sure, I'll change it.